contrib/join-library/src/main/java/com/google/cloud/dataflow/contrib/joinlibrary/Join.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (C) 2014 Google Inc.
```

Comment:
I'd suggest changing this to: `Copyright (C) 2015 The Google Cloud Dataflow Authors`, here and in other `*.java` files, to acknowledge your contributions. If checkstyle complains, feel free to fix the regex (or I can do it).